pkgs/servers/tracing/tempo/default.nix

```diff
@@ -2,23 +2,22 @@
 
 buildGoModule rec {
   pname = "tempo";
-  version = "1.4.1";
+  version = "1.5.0";
 
   src = fetchFromGitHub {
     owner = "grafana";
     repo = "tempo";
     rev = "v${version}";
     fetchSubmodules = true;
-    sha256 = "sha256-kxR+xwhthsK3gThs0jPJfWlsRG35kCuWvKH3Wr7ENTs=";
+    sha256 = "sha256-m7tfDd0Yjg4+VHZPxYJXEx2XNNodepMcPLucBjvd88s=";
   };
 
   vendorSha256 = null;
 
   subPackages = [
     "cmd/tempo-cli"
     "cmd/tempo-query"
-    # FIXME: build is broken upstream, enable for next release
-    # "cmd/tempo-serverless"
+    "cmd/tempo-serverless"
```

Comment:
This is not installed as far as I can tell. Should there be a separate binary?

Reply:
No, it's example code, but it's worth building to make sure it's correct.